.github/workflows/integration_tests.yml

```
@@ -174,5 +174,7 @@ jobs:
         run: docker exec -t dev_rucio_1 tools/pytest.sh -v --tb=short tests/test_did_meta_plugins.py::TestDidMetaMongo
       - name: Test external metadata plugin (postgres)
         run: docker exec -t dev_rucio_1 tools/pytest.sh -v --tb=short tests/test_did_meta_plugins.py::TestDidMetaExternalPostgresJSON
+      - name: Test external metadata plugin (elasticsearch)
+        run: docker exec -t dev_rucio_1 tools/pytest.sh -v --tb=short tests/test_did_meta_plugins.py::TestDidMetaElasticsearch
```

The CI is failing because it cannot find the tests: https://github.com/rucio/rucio/actions/runs/11408355858/job/32225753284

The `yml` file here is using the wrong class name, it should be:

```suggestion
        run: docker exec -t dev_rucio_1 tools/pytest.sh -v --tb=short tests/test_did_meta_plugins.py::TestDidMetaElastic
```

(or the class name should be changed to `TestDidMetaElasticsearch`)